src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2SecurityTokenHandler.ValidateToken.Internal.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+namespace Microsoft.IdentityModel.Tokens.Saml2
+{
+    /// <summary>
+    /// A <see cref="SecurityTokenHandler"/> designed for creating and validating Saml2 Tokens. See: http://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf
+    /// </summary>
+    public partial class Saml2SecurityTokenHandler : SecurityTokenHandler
+    {
+
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
are these pragma warning disable's temporary? I assume they will be removed once the rest of the code is complete?

Reply:
Correct, they're temporary for now until we get to those parts of the flow.

Reply:
Sure, I created it [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2880).